### PR TITLE
[13.0][MIG] account_check_deposit

### PR DESCRIPTION
**Pull request by ps-tubtim**
Standard migration.

### CODE REVIEW

**Review by alexis-via — APPROVED** [review]
code review, no test

File: account_check_deposit/security/check_deposit_security.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    @author Alexis de Lattre <alexis.delattre@akretion.com>
+    The licence is in the file __manifest__.py
+-->
+<odoo noupdate="1">
+    <record id="check_deposit_rule" model="ir.rule">
+        <field name="name">Check Deposit multi-company</field>
+        <field name="model_id" ref="model_account_check_deposit" />
+        <field
+            name="domain_force"
+        >['|', ('company_id', '=', False), ('company_id', 'child_of', [user.company_id.id])]</field>
```
Comment (ivantodorovich) [inline account_check_deposit/security/check_deposit_security.xml]:
```suggestion
        <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
```

https://www.odoo.com/documentation/13.0/howtos/company.html#security-rules

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Minor comment, otherwise LGTM :+1:

File: account_check_deposit/security/check_deposit_security.xml (hunk 1 above)
Comment (ps-tubtim) [reply account_check_deposit/security/check_deposit_security.xml]: Thank you for review. I edited this code.

**Comment by ivantodorovich** [comment]
Pre-commit is failing on that last one. Have you installed it in the repo?

**Comment by ps-tubtim** [comment]
@ivantodorovich I fixed pre-commit, all checks have passed.

**Review by ivantodorovich — APPROVED** [review]

**Pull request merged**